nix/sources.json

```diff
@@ -34,5 +34,17 @@
         "type": "tarball",
         "url": "https://github.com/nix-community/poetry2nix/archive/f0cc43e09f6adbbde8c1207d511cd124cdca28f4.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
+    },
+    "relayer": {
```

Comment:
is it because of this patch: https://github.com/yihuang/relayer/commit/5e9fd91198e206fe85be6214da68f4250b557063 ?
can the Rust relayer do it: https://github.com/informalsystems/ibc-rs/ ?

Reply:
yes, did't tried ibc-rs yet, let me try try.

Reply:
https://github.com/informalsystems/ibc-rs/blob/master/relayer/src/keyring/store.rs#L108
it also hard coded the hd path.